internal/links/provider.go

```diff
@@ -123,3 +123,34 @@ func renderLink(linkTemplate string, data any) (string, error) {
 	}
 	return message.String(), nil
 }
+
+func resolveProjectTemplates(project *v1alpha1.Project, catalogProjectLinks map[string]string) map[string]string {
```

Comment:
The power of `maps`!
```go
links := maps.Clone(catalogProjectLinks)
maps.Copy(links, project.Spec.Links)
return links
```

Reply:
Magnifique, TIL! 👍
Cependant, j'ai fini par utiliser juste des `.Copy`, car malheureusement `maps.Clone(nil)` retourne nil, ce qui a du sens en règle général, mais pas dans mon cas, car ça se retrouvait à essayer d'écrire dans une map nil. Alors donc:
```go
links := make(map[string]string)
maps.Copy(links, catalogProjectLinks)
maps.Copy(links, project.Spec.Links)
return links
```